scripts/maintenance/computational-clusters/osparc_clusters.py

```diff
@@ -75,9 +75,9 @@ class DynamicInstance(AutoscaledInstance):
 MINUTE: Final[int] = 60
 HOUR: Final[int] = 60 * MINUTE
 
-DEFAULT_COMPUTATIONAL_EC2_FORMAT: Final[str] = (
-    r"osparc-computational-cluster-{role}-{swarm_stack_name}-user_id:{user_id:d}-wallet_id:{wallet_id:d}"
-)
+DEFAULT_COMPUTATIONAL_EC2_FORMAT: Final[
+    str
+] = r"osparc-computational-cluster-{role}-{swarm_stack_name}-user_id:{user_id:d}-wallet_id:{wallet_id:d}"
```

Comment:
I would actually prefer the previous formatting here. I find it strange that the `str` should sit on its own line. But I guess black is to blame... 🤔